pkg/syncer/util.go

```diff
@@ -230,15 +230,15 @@ func isValidvSphereVolumeClaim(ctx context.Context, pvcMetadata metav1.ObjectMet
 	// Checking if the migrated-to annotation is found in the PVC metadata.
 	if annotation, annMigratedToFound := pvcMetadata.Annotations[common.AnnMigratedTo]; annMigratedToFound {
 		if annotation == csitypes.Name &&
-			pvcMetadata.Annotations[common.AnnStorageProvisioner] == common.InTreePluginName {
+			pvcMetadata.Annotations[common.AnnBetaStorageProvisioner] == common.InTreePluginName {
```

Comment:
This code is for vanilla only. Revert this change.

Reply:
Just a variable name change.